src/products/README.md

```diff
@@ -40,3 +40,39 @@ Updates the current inventory value for the product identified by `{productID}`.
 Returns details on all categories.
 ### GET /categories/id/{categoryID}
 Returns details on the category identified by `{categoryID}`.
+
+## Testing
+To run integration tests for the Products service a Python virtual environment is required. You must have Python 3.3+ installed on your system to run the commands below. The commands are written to be ran from the project root.
+
+The following command will create a virtual environment in the test directory of the products service. 
+```console
+python3 -m venv src/products/test/.venv
```

Comment:
(nit) May be we tell users upfront that we will run all of this in `src/products/test` so that we don't need to put prefix for every line?